src/components/RichContenteditable/RichContenteditable.vue

```diff
@@ -441,20 +456,20 @@ export default {
 // Standalone styling, independent from server
 .rich-contenteditable__input {
 	overflow-y: auto;
-	width: auto;
+	width: 100%;
 	margin: 0;
-	padding: 6px;
+	padding: 8px;
 	cursor: text;
 	white-space: pre-wrap;
 	word-break: break-word;
 	color: var(--color-main-text);
 	border: 1px solid var(--color-border-dark);
-	border-radius: var(--border-radius);
+	border-radius: var(--border-radius-large);
```

Comment:
Please add screenshots as this is a design change

Reply:
Sure

![Screenshot from 2020-11-11 11-52-06](https://user-images.githubusercontent.com/26852655/98802961-5f322600-2414-11eb-9491-398c8a08b5b2.png)